gnupg-pkcs11-scd/command.c

```diff
@@ -1132,7 +1132,7 @@ gpg_error_t _cmd_pksign_type (assuan_context_t ctx, char *line, int typehint)
 	/*
 	 * sender prefixed data with algorithm OID
 	 */
-	if (strcmp(hash, "")) {
+	if (hash != NULL && strcmp(hash, "")) {
```

Comment:
correct! should only be checked against null, I will modify.

Reply:
Seems to work, thanks!